README.md

```diff
@@ -31,22 +31,38 @@ cd osparc-ops
 make help
 ```
 
+### WSL deployement
+
+If you are using WSL1, clone the repository under the Windows filesystem and access it in WSL. You need to ensure [Ensure Volume Mounts Work](https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly) (Section Volume Mounts Work)
+
+Do not clone the repo directly in your WSL1 filesystem. 
+
+eg : Do not clone here
+```console 
+pwd
+/home/alexandre
+```
+But here
+```console 
+pwd
+/c/Users/Alexandre/
+```
+
 #### local deployment
   ```console
   make up-local
   ```
 A self-signed certificate may be generated. The system host file may be modified using the default **osparc.local** fully qualified domain name (FQDN) to point to the local machine.
 
 The services above will be deployed and pre-configured on the following endpoints:
-  - Traefik: [https://osparc.local:9001/dashboard/](https://osparc.local:9001/dashboard/)
-  - Portainer: [https://osparc.local/portainer/](https://osparc.local/portainer/)
-  - Minio: [https://osparc.local/minio](https://osparc.local/minio) and [https://osparc.local:10000](https://osparc.local:10000)
-  - Portus: [https://osparc.local:5000](https://osparc.local:5000)
+  - Traefik: [https://monitoring.osparc.local/dashboard/#/](https://monitoring.osparc.local/dashboard/#/)
+  - Portainer: [https://monitoring.osparc.local/portainer/](https://monitoring.osparc.local/portainer/)
+  - Minio: [https://storage.osparc.local/minio/](https://storage.osparc.local/minio/)
```

Comment:
why is there still /minio here? in our deploys it would be just https://storage.osparc.local

Reply:
https://storage.osparc.local automatically redirect to https://storage.osparc.local/minio

Reply:
ok